.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: Build
+on:
+  workflow_call:
+
+env:
+  BUILD_ROOT: ${{ github.workspace }}/bazel-bin/build
```

Comment:
instead of hard code here, is it possible to parameterize this? so that we don't need to duplicate in multiple yamls (and in the future perf.yml can also updated to use this file as well).

Reply:
that should be easy, thanks for the suggestion